fails to build gnutls (possibly undefined macro: AC_MSG_ERROR)

Hello,

building gnutls on Debian/sid fails on both stable and master branch, with:

```
[build]    gnutls

Failed to build package gnutls!
------------------------------------------------------------
configure.ac:209: error: possibly undefined macro: AC_MSG_ERROR
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
Makefile:374: recipe for target 'build-only-gnutls_i686-pc-mingw32.static' failed
make[1]: *** [build-only-gnutls_i686-pc-mingw32.static] Error 1
make[1]: Leaving directory '/home/eroux/softs/mxe'
real    0m13.902s
user    0m13.624s
sys 0m0.216s
------------------------------------------------------------
[log]      /home/eroux/softs/mxe/log/gnutls
```

Is there a simple workaround?

--- a/src/gnutls.mk
+++ b/src/gnutls.mk
@@ -19,9 +19,7 @@ endef
 
 define $(PKG)_BUILD
     $(SED) -i 's, sed , $(SED) ,g' '$(1)/gl/tests/Makefile.am'
-    cd '$(1)' && aclocal -I m4 -I gl/m4 -I src/libopts/m4 --install
-    cd '$(1)' && autoconf
-    cd '$(1)' && automake --add-missing
+    cd '$(1)' && autoreconf -fi -I m4 -I gl/m4 -I src/libopts/m4
     # skip the run test for libregex support since we are cross compiling
     $(SED) -i 's/libopts_cv_with_libregex=no/libopts_cv_with_libregex=yes/g;' '$(1)/configure'
     # AI_ADDRCONFIG referenced by src/serv.c but not provided by mingw.

